### ISSUE TITLE
GIoU Loss

### ISSUE THREAD

**Issue by nguyenthuy1681999** [sharad5/OWL-ViT-Object-Detection#2]
`  def get_giou_loss(self, pred_boxes, target_boxes, step):
        giou_loss = -1.0 * BoxUtil.generalized_box_iou(pred_boxes, target_boxes, step)
        return giou_loss
`
I don't understand why the GIoU loss function is used to multiply by -1.0 instead of using the original function  BoxUtil.generalized_box_iou(pred_boxes, target_boxes, step)
I think it GIoU loss function:
`giou_loss = 1.0 - BoxUtil.generalized_box_iou(pred_boxes, target_boxes, step)`
and **get_mean_non_overlap_loss**:
`non_overlapping_mask = giou_loss < self.non_overlap_giou_thresh` with non_overlap_giou_thresh = 0.5
Please explain to me. Thank you.
